app/controllers/logs_controller.rb

```diff
@@ -44,7 +44,7 @@ def tardy
   end
 
   def index(logs=nil, header='Entire Log')
-    filter = filter.nil? ? '' : " AND #{filter}"
+    filter.nil? ? '' : " AND #{filter}"
```

Comment:
@rylanb I was also confused, it was flagged for not being used, so I deleted the assignment. It seems like if it's not used, this line could be deleted, but I wasn't sure.